pkg/logs/status/status.go

```diff
@@ -78,22 +79,25 @@ func Get() Status {
 				Configuration: toDictionary(source.Config),
 				Status:        status,
 				Inputs:        source.GetInputs(),
-				Overview:      source.Overview,
+				Messages:      source.Messages.GetMessages(),
 			})
+
+			for _, warning := range source.Messages.GetWarnings() {
+				warningsDeduplicator[warning] = true
+			}
 		}
 		integrations = append(integrations, Integration{Name: name, Sources: sources})
 	}
 
-	warnings := GetWarnings()
-	warningMessages := make([]string, len(warnings))
-	for i, w := range warnings {
-		warningMessages[i] = w.Render()
+	warnings := make([]string, 0)
+	for warning := range warningsDeduplicator {
+		warnings = append(warnings, warning)
 	}
 
 	return Status{
-		IsRunning:       true,
-		Integrations:    integrations,
-		WarningMessages: warningMessages,
+		IsRunning:    true,
+		Integrations: integrations,
+		Messages:     warnings,
```

Comment:
I think this is a bit misleading, I am not sure the messages that we want to report at the very beginning of the logs status are always related to logs-sources.
We could for example report to the user that he is using a deprecated attribute and the he should use the new one.